### PR TITLE
Feat: Upgrade AJV from 6 to 8

### PR DESCRIPTION
**Pull request by vwong**
* much of validate-json has been gutted out, and replaced with [ajv-formats](https://github.com/ajv-validator/ajv-formats)
* many of the error strings have been updated
* int64, int32, float validation has been relaxed according to https://github.com/ajv-validator/ajv-formats/pull/22
* `nullable` requires use of `type`, see https://github.com/ajv-validator/ajv/blob/490eb8c0eba8392d071fef005e16d330f259d0ba/lib/compile/validate/dataType.ts#L26C66-L26C66
* other packages get a bump too, because why not?

### CODE REVIEW

File: lib/swagger-mock-validator/spec-parser/parsed-spec.d.ts [hunk 1]
```diff
@@ -60,8 +60,8 @@ interface ParsedSpecJsonSchemaBooleanKeywords {
 interface ParsedSpecJsonSchemaValue {
     additionalProperties?: boolean | ParsedSpecJsonSchema;
     enum?: any[];
-    exclusiveMaximum?: boolean;
-    exclusiveMinimum?: boolean;
+    exclusiveMaximum?: number;
+    exclusiveMinimum?: number;
```
Comment (vwong) [inline lib/swagger-mock-validator/spec-parser/parsed-spec.d.ts:64]: OpenAPI defines these as booleans, as did AJV@6.

But since https://json-schema.org/draft-06/json-schema-release-notes.html, it has been converted to a number. AJV followed suite.

File: test/unit/formats.spec.ts [hunk 2]
```diff
@@ -356,9 +376,7 @@ describe('formats', () => {
             .withParameter(pathParameterBuilder.withInt64Named('value'));
 
         const minimumInt64Allowed = '-9223372036854775808';
-        const minimumInt64AllowedMinusOne = '-9223372036854775809';
         const maximumInt64Allowed = '9223372036854775807';
-        const maximumInt64AllowedPlusOne = '9223372036854775808';
```
Comment (vwong) [inline test/unit/formats.spec.ts:361]: See https://github.com/ajv-validator/ajv-formats/pull/22#issuecomment-808896823

TLDR: JSON only supports doubles, and JS (without resorting to BigInt) too; pretending otherwise is pointless.

**Review by bethesque — APPROVED** [review]
🤷🏽‍♀️ I trust you Voon!

**Review by Inksprout — APPROVED** [review]
